test/e2e/user_management_test.go

```diff
@@ -42,7 +45,99 @@ func (s *userManagementTestSuite) TearDownTest() {
 	s.testCtx.Cleanup()
 }
 
+func (s *userManagementTestSuite) TestUserDeactivation() {
+	s.setApprovalPolicyConfig("automatic")
+
+	userSignup, mur := s.createAndCheckUserSignup(true, "iris-at-redhat-com", "iris@redhat.com", approvedByAdmin()...)
+
+	// Deactivate the user
+	userSignup.Spec.Deactivated = true
+	err := s.awaitility.Client.Update(context.TODO(), userSignup)
+	require.NoError(s.T(), err)
+	s.T().Logf("user signup '%s' set to deactivated", userSignup.Name)
+
+	err = s.hostAwait.WaitUntilMasterUserRecordDeleted(mur.Name)
+	require.NoError(s.T(), err)
+}
+
+func (s *userManagementTestSuite) TestUserBanning() {
+
+	s.T().Run("user banning", func(t *testing.T) {
+		// when
+		s.setApprovalPolicyConfig("automatic")
+
+		// then
+		s.checkUserBanned()
```

Comment:
could you please replace this call of the `checkUserBanned` method by the actual subtests? The current state is confusing.

Reply:
Certainly, I was just copying the structure in `usersignup_test` which does exactly this, however I've got no problem putting the subtests into one single test method.

Reply:
Thanks, it looks much better now :+1: